opengate/actors/doseactors.py

```diff
@@ -625,6 +639,12 @@ def initialize(self, *args):
         # C++ side has a boolean toWaterFlag and self.score_in == "water" yields True/False
         self.SetToWaterFlag(self.score_in == "water")
 
+        # variables for stop on uncertainty functionality
+        self.SetUncertaintyGoal(self.uncertainty_goal)
```

Comment:
As a consequence of my other comment, I would add a little logic here: 

```python
if self.uncertainty_goal is None: 
   self.SetUncertaintyGoal(0)
else: 
   self.SetUncertaintyGoal(self.uncertainty_goal)
```
In this way, we can use 0 as indicator on the C++ side that no goal is requested, but use the more user-friendly None on the python side.